docs/guide/agent-server-compatibility.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[agent-server-compatibility]]
+== Agent/Server Compatibility
+
+Below is a chart that outlines the compatibility between different versions of the APM agents and the APM Server. 
+
+NOTE: APM Server version 6.5 introduced a new way for APM agents to communicate with the APM Server.
+To take advantage of new features,
+it is recommended that you update to the newest version of the applicable agent.
+
+[float]
+[[nodejs-compatibility]]
+==== Node.js Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
```

Comment:
I can see that you've documented support fro pre-v6.2 for some of the other agents, so just wanted to say that version 0.x of the Node agent supported version <6.2 of the APM Server

Reply:
Sorry to catch up on this late - do we really want to officially document pre-release candidates and their compatibility with the server? I'd start documenting the server from `6.2` on and agents from when they were GA .

Reply:
I also think we should not document 0.x agents as it indicates we still support them which I don't think we do.

Reply:
I don't think the concept of what 0.x in the agents are aligned to what we support or not. Each agent uses their own versioning scheme, so simply saying that we shouldn't document 0.x would be wrong. But I'm fine with documenting only support for 6.2 and up. I just wanted to mention that you can't see that from the agent version number :)

Reply:
> start documenting the server from 6.2 on and agents from when they were GA .

My suggestion was to document Server versions from 6.2 (GA) on and agent versions from their first stable version on. Correct me if I am wrong, but I assume it is standard between languages that `0.x` versions are not considered to be stable.

Reply:
@simitt that's ok. I was just commenting on Felix comment about 0.x being not support in general